src/index.tsx

```diff
@@ -220,7 +229,13 @@ export default class ReactToPrint extends React.Component<IReactToPrintProps> {
             nonce,
         } = this.props;
 
-        const contentEl = content();
+        let contentEl = lazyOption && typeof lazyOption.content === "function" ? lazyOption.content() : null;
+        
+        if (!contentEl) {
+            if (typeof content === "function") {
+                contentEl = content();
+            }
+        }
```

Comment:
I believe `content` is already typed as a function at this point?
```suggestion
        if (!contentEl && content) {
                contentEl = content();
        }
```

Reply:
This looks good @MatthewHerbst ! Thanks for the suggestion (collapsing into a single `if`). I will apply it